universal-login-sdk/lib/sdk.ts

```diff
@@ -162,6 +161,11 @@ class UniversalLoginSDK {
     this.deploymentObserver = this.deploymentObserver || new DeploymentObserver(this.blockchainService, this.config!.contractWhiteList);
   }
 
+  private getFutureWalletFactory() {
```

Comment:
This one could be actually created in constructor

Reply:
In constructor we don't have RelayerConfig yet